pkg/manifest/raw.go

```diff
@@ -59,6 +61,38 @@ func (p *RawImage) serialize() osbuild.Pipeline {
 	copyInputs := osbuild.NewPipelineTreeInputs(inputName, p.treePipeline.Name())
 	pipeline.AddStage(osbuild.NewCopyStage(copyOptions, copyInputs, copyDevices, copyMounts))
 
+	bootFiles := p.treePipeline.platform.GetBootFiles()
+	if len(bootFiles) > 0 {
+		// we ignore the bootcopyoptions as they contain a full tree copy instead we make our own, we *do* still want all the other
+		// information such as mountpoints and devices
+		_, bootCopyDevices, bootCopyMounts := osbuild.GenCopyFSTreeOptions(inputName, p.treePipeline.Name(), p.Filename(), pt)
+		bootCopyOptions := &osbuild.CopyStageOptions{}
+		bootCopyInputs := osbuild.NewPipelineTreeInputs(inputName, p.treePipeline.Name())
+
+		// Find the FS root mount name to use as the destination root
+		// for the target when copying the boot files.
+		var fsRootMntName string
+		for _, mnt := range bootCopyMounts {
+			if mnt.Target == "/" {
+				fsRootMntName = mnt.Name
+				break
+			}
+		}
+
+		if fsRootMntName == "" {
+			panic("no mount found for the filesystem root")
```

Comment:
Shouldn't panic in a library.

Reply:
Yea, this was a quick copy and paste to unblock (from `raw_ostree`), we'll be removing all of these panics.